zproject/dev_urls.py

```diff
@@ -104,7 +105,7 @@
     ]
 else:
 
-    def serve_static(request: HttpRequest, path: str) -> HttpResponse:
+    def serve_static(request: WSGIRequest, path: str) -> FileResponse:
```

Comment:
`FileResponse` makes sense (it’s an `HttpResponseBase` but not an `HttpResponse`), but `request` should remain `HttpRequest`.